### PR TITLE
Mwpw 143327 app banner

### PR DESCRIPTION
**Pull request by vhargrave**
Resolves: [MWPW-143327](https://jira.corp.adobe.com/browse/MWPW-143327)

To test you'll have to test on mobile. I did a crawl and this is the only live page with the app-banner.

Test URLs:
- Pre Migration Link: https://adobe.com/express/
- Before: https://main--express-milo--adobecom.hlx.page/jp/express/create/advertisement/facebook
- After: https://MWPW-143327-app-banner--express-milo--adobecom.hlx.page/jp/express/create/advertisement/facebook

### CODE REVIEW

File: express/blocks/app-banner/app-banner.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    localStorage.setItem('app-banner-optout-exp-date', sevenDaysFromNow);
+
+    setTimeout(() => {
+      $background.remove();
+    }, 600);
+  });
+}
+
+function initScrollDirection(block) {
+    debugger;
```
Comment (fullcolorcoder) [inline express/blocks/app-banner/app-banner.js:98]: Debugger

File: express/blocks/app-banner/app-banner.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  const $section = block.closest('.section');
```
Comment (fullcolorcoder) [inline express/blocks/app-banner/app-banner.js:99]: linting

**Review by fullcolorcoder — CHANGES_REQUESTED** [review]
Text isn't aligned the same as prod. 

Prod: 
image

branch: 
image

- What are all the other files for? Are there additional planned changes aside from app banner?

**Comment by vhargrave** [comment]
> Text isn't aligned the same as prod.
> 
> Prod: image
> 
> branch: image
> 
> * What are all the other files for? Are there additional planned changes aside from app banner?

@fullcolorcoder yeah, I know the text is a bit confusing. I should have written something. The text is getting formatted because of milo's formatting of JP texts. As I've agreed to with the JP team, we'll allow milo to do formatting of any h1-h6 tags, even if they look different. That's what's happening here. 
I've removed the debugger statement thanks! 
Finally, for the files you were seeing, that was because I had branched off the "general-fixes" branch to have a less buggy version of everything. Now that that's merged though, you won't see the changes anymore.

**Review by fullcolorcoder — APPROVED** [review]

**Pull request merged**